x/participation/module_simulation.go

```diff
@@ -47,7 +47,7 @@ func (AppModule) ProposalContents(_ module.SimulationState) []simtypes.WeightedP
 
 // RandomizedParams creates randomized  param changes for the simulator
 func (am AppModule) RandomizedParams(_ *rand.Rand) []simtypes.ParamChange {
-	participationParams := types.DefaultParams()
+	participationParams := sample.ParticipationParams()
```

Comment:
See my comment above, randomness would be good for this part but not blocking for this PR

Reply:
I think there's a misunderstanding. params addressed in this PR *are* randomized

Reply:
same as above